wdn/templates_5.1/js-src/slideshows.babel.js

```diff
@@ -0,0 +1,124 @@
+require(['dcf-utility', 'dcf-slideshow', 'css!js-css/slideshows'], () => {
+  const openCaptionEvent = new Event('openCaption');
+  const closeCaptionEvent = new Event('closeCaption');
+  const slideshows = document.querySelectorAll('.dcf-slideshow');
+  const uls = document.querySelectorAll('.dcf-slideshow ul');
+
+  const slideshow = new DCFSlideshow(slideshows, uls, openCaptionEvent, closeCaptionEvent);
+
+  slideshow.initialize();
+
+  const keyframesClose1 = [
+    {
+      transform: 'rotate(45deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const keyframesClose2 = [
+    {
+      transform: 'rotate(-45deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const keyframesOpen1 = [
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(45deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const keyframesOpen2 = [
+    {
+      transform: 'rotate(0deg)',
+      transformOrigin: '50% 50%'
+    },
+    {
+      transform: 'rotate(-45deg)',
+      transformOrigin: '50% 50%'
+    }
+  ];
+  const options = {
+    duration: 250,
+    fill: 'forwards'
+  };
+
+  Array.prototype.forEach.call(slideshows, (slideshow) => {
+    // Slideshow Controls
+    let ctrlGroup = slideshow.querySelector('.slide-show-control'); // DCF
+    let ctrls = ctrlGroup.querySelectorAll('li');
+
+    Array.prototype.forEach.call(ctrls, (ctrl) => {
+      if (ctrl.classList.contains('slide-prev-btn')) {
+        let ctrlButton = ctrl.querySelector('button');
+        ctrlButton.classList.add('dcf-d-flex', 'dcf-ai-center', 'dcf-pt-4', 'dcf-pb-4', 'dcf-inverse');
+        ctrlButton.innerHTML =
+          '<svg class="dcf-h-4 dcf-w-4 dcf-fill-current" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true">' +
+          '<path d="M21.746.064a.504.504 0 0 0-.504.008l-19 11.5a.499.499 0 0 0-.001.856l19' +
+          ' 11.5A.501.501 0 0 0 22 23.5V.5a.5.5 0 0 0-.254-.436z"></path>' +
+          '</svg>';
+      } else if (ctrl.classList.contains('slide-next-btn')) {
+        let ctrlButton = ctrl.querySelector('button');
+        ctrlButton.classList.add('dcf-d-flex', 'dcf-ai-center', 'dcf-pt-4', 'dcf-pb-4', 'dcf-inverse');
+        ctrlButton.innerHTML =
+          '<svg class="dcf-h-4 dcf-w-4 dcf-fill-current" width="24" height="24" viewBox="0 0 24 24" aria-hidden="true">' +
+          '<path d="M21.759 11.577L2.786.077a.499.499 0 0 0-.759.428v23a.498.498 ' +
+          '0 0 0 .5.5c.09 0 .18-.024.259-.072l18.973-11.5a.5.5 0 0 0 0-.856z"></path>' +
```

Comment:
I've updated the next arrow icon: `'<path d="M23.852 11.144L13.703 1.096c-.96-.96-2.678-.924-3.68.075-1.036 1.035-1.07 2.687-.069 3.69l4.321 4.143H2.03c-1.27 0-2.03 1.272-2.03 2.5 0 .617.168 1.207.472 1.659.369.549.913.851 1.53.851h12.276l-4.156 4.16a2.425 2.425 0 00-.708 1.734c0 .708.293 1.409.807 1.922a2.738 2.738 0 001.919.806c.664 0 1.28-.251 1.739-.708l9.977-10.076a.502.502 0 00-.004-.708z"></path>' +`